### DISCUSSION TITLE
Where is the changelog for 0.4.1?

### DISCUSSION THREAD

**Discussion by Tejeev** [aptend/typing-transformer-obsidian#83] [Q&A] [answered]
I see 'bugfix' but can't find any info on what issue was addressed.  I know I can look at the code, but once I figure that out could I update the changelog to be a little more verbose if I can make sense of the code change?

**Comment by aptend** [comment, marked as answer]
Do you mean you'd like to help with the changelog? That's great
In 0.4.1
1. 4 bugs were fixed, #72 #76 #78 #74. 
2. A Debug option was added to the settings page, controlling the log output to the console
3. Disable the zoneIndictor feature by default.

**Reply by Tejeev** [reply]
Sure :) 
How do I add these notes to the release notes (https://github.com/aptend/typing-transformer-obsidian/releases/tag/0.4.1)?

And thank you! that's exactly the info I was looking for.  Do you have it stored somewhere and I just didn't see it?

**Reply by Tejeev** [reply]
(I tried to see if I could like fork edit and PR the release notes but didn't see a way in the UI.  I suppose I should clone the repo and see if i can find the file for the release notes and edit it directly?)

**Reply by aptend** [reply]
I'm inviting you as a collaborator and I guess then you will be able to edit the changelog directly

**Reply by Tejeev** [reply]
Yep! I was able to add the changes directly!
I'm not a programmer and my skills are very Rancher/K8s specific but I'll be happy to help in any way I can :)

**Reply by aptend** [reply]
I would greatly appreciate any help!
